src/content/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/opentelemetry-setup.mdx

```diff
@@ -285,30 +285,34 @@ Whether you export directly from your app or from a collector, you'll need to:
   </tbody>
 </table>
 
-<Callout variant="important">
-  In Node.js, the [opentelemetry-collector-exporter-grpc](https://www.npmjs.com/package/@opentelemetry/exporter-trace-otlp-grpc) library requires additional options to enable TLS.
-</Callout>
-
 <CollapserGroup>
   <Collapser
     className="freq-link"
     id="note-ports"
-    title="Additional information about ports"
+    title="Comparison of FedRAMP and non-FedRAMP endpoints"
   >
-    OTLP standards designate gRPC traffic to port `4317`, and HTTP traffic to port `4318`. The New Relic US FedRamp Native OTLP endpoint adheres to those specifications, as well as allowing gRPC traffic on port `443`.
+    OTLP standards designate gRPC traffic to port `4317`, and HTTP traffic to port `4318`.
 
-    However, non-FedRamp New Relic endpoints will accept both gRPC and HTTP traffic on any of the ports listed in the above chart.
-  </Collapser>
+    The New Relic US FedRamp Native OTLP endpoint adheres to those specifications, as well as allowing gRPC traffic on port `443`. For FedRAMP accounts, the OTLP Metrics integration **only** supports the HTTP endpoint. See [FedRAMP compliance](/docs/security/security-privacy/compliance/fedramp-compliant-endpoints/#otlp-api) for more details.
```

Comment:
super minor thing, but sometimes the 'only' is modifying wrong word. i'd put 'supports only...' instead of 'only supports...'

Reply:
@zuluecho9 nice suggestion!